x-pack/plugins/snapshot_restore/public/application/components/repository_form/step_one.tsx

```diff
@@ -237,18 +237,17 @@ export const RepositoryFormStepOne: React.FunctionComponent<Props> = ({
           repositoryTypes.includes(REPOSITORY_TYPES.url) ? (
             <FormattedMessage
               id="xpack.snapshotRestore.repositoryForm.fields.defaultTypeDescription"
-              defaultMessage="Elasticsearch supports file system and read-only URL repositories.
-                Additional types require plugins. {docLink}"
+              defaultMessage="Storage location for your snapshots. Additional repository types are available as plugins. {docLink}"
```

Comment:
Note to reviewers:

I'm not 100% convinced we need separate cloud and default messages anymore. There's only one repository type left (Hadoop), and it may not warrant a separate sentence here. However, I'd love some feedback before removing it.

Reply:
I agree with you that we don't necessarily need a separate message for Cloud, especially with the link now leading to the repository types doc page (not the plugins page).